### PR TITLE
profile page

### PR DESCRIPTION
**Pull request by aayush017**

### CODE REVIEW

File: lib/data/user.dart [hunk 1]
```diff
@@ -17,9 +18,12 @@ class UserProfile {
       username: json["username"],
       profileImage: json["imageUrl"],
       pronouns: json["pronouns"],
+      points: 100,
```
Comment (antonio-pedro99) [inline lib/data/user.dart]: Why starting with 100?
We do not need this. All the users will start with 0 points.

File: lib/view/screens/profile/create_profile_screen.dart [hunk 2]
```diff
@@ -41,15 +40,15 @@ class _CreateProfileScreenState extends State<CreateProfileScreen> {
 
   void _saveProfile(BuildContext context) {
     if (_profileImage != null) {
-
       UserProfile user = UserProfile(
         fullName: nameTextEditingController.text,
         pronouns: pronounsTextEditingController.text,
         username: usernameTextEditingController.text,
         profileImage: _profileImage!.path,
+        points: 100,
```
Comment (antonio-pedro99) [inline lib/view/screens/profile/create_profile_screen.dart]: starting from 0, no 100

File: lib/view/screens/profile/profile_screen.dart [hunk 3]
```diff
@@ -1,19 +1,81 @@
...
     return Scaffold(
-      appBar: AppBar(
-        title: Text('Profile Page'),
-      ),
-      body: Center(
-        child: Text(
-          'This is the Profile Page',
-          style: TextStyle(fontSize: 24),
-        ),
+      appBar: AppBar(),
```
Comment (antonio-pedro99) [inline lib/view/screens/profile/profile_screen.dart:17]: why an empty AppBar?

**Review by antonio-pedro99 — CHANGES_REQUESTED** [review]
Solid work @aayush017 !
Just left minors suggestions.

File: lib/data/user.dart (hunk 1 above)
Comment (aayush017) [reply lib/data/user.dart]: ig this will be stored in the firebase so you can set it according to your pref ie 0

Comment (antonio-pedro99) [reply lib/data/user.dart]: First data will come from the phone hahaha.

**Review by antonio-pedro99 — APPROVED** [review]
LGTM

**Pull request merged**